projects/admin/src/app/circulation/patron/card/card.component.html

```diff
@@ -39,7 +39,7 @@ <h3>
       <div class="row">
         <div class="col-md-6">
           <div>
-            <span id="patron-birth-date" *ngIf="patron.birth_date">{{ patron.birth_date | date :'shortDate' }}</span>
+            <span id="patron-birth-date" *ngIf="patron.birth_date">{{ patron.birth_date | dateTranslate:'mediumDate' }}</span>
```

Comment:
Why change 'shortDate' to 'mediumDate' ? In the app, we always use the short (YYYY-MM-DD) instead of Medium (MMM d, y) ?